tests/HasStatusesTest.php

```diff
@@ -180,4 +180,24 @@ public function it_can_return_a_string_when_calling_the_attribute()
 
         $this->assertEquals('waiting for a change', $this->testModel->status()->reason);
     }
+
+    /** @test */
+    public function it_can_find_all_models_that_do_not_have_a_status_with_the_given_name()
+    {
+        $model1 = TestModel::create(['name' => 'model1']);
```

Comment:
Could let this test prove that a model without a status will be found by the scope as well?

Reply:
@freekmurze Cool, I added an or condition to also look for all models w/o a status

is this what you were thinking?